deployments/qat_plugin/base/intel-qat-plugin.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: intel-qat-plugin
   labels:
     app: intel-qat-plugin
+  annotations:
+    container.apparmor.security.beta.kubernetes.io/intel-qat-plugin: unconfined
```

Comment:
DaemonSet annotations are also not needed

Reply:
@mythi
This also need to be present to have it as a default.